ui/scripts/vpc.js

```diff
@@ -436,6 +401,38 @@
                 isOptional: true,
                 isEditable: true
             },
+            'icmptype': {
+                edit: true,
+                label: 'ICMP.type',
+                desc: 'Please specify -1 if you want to allow all ICMP types',
```

Comment:
How about adding internationalization to desc field? (the same for icmpcode below)

Reply:
Done